packages/conjure-client/package.json

```diff
@@ -25,31 +25,34 @@
     "@types/chai": "^4.1.7",
     "@types/express": "^4.16.1",
     "@types/fetch-mock": "^5.12.2",
-    "@types/jest": "^21.1.10",
+    "@types/jest": "^24.0.23",
     "chai": "^4.2.0",
     "es6-shim": "^0.35.5",
     "express": "^4.16.4",
     "fetch-mock": "^5.13.1",
-    "jest": "^21.2.1",
+    "jest": "^24.9.0",
     "karma": "^2.0.5",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^1.3.0",
     "karma-nodeunit": "^0.2.0",
-    "karma-typescript": "^3.0.13",
+    "karma-typescript": "^4.1.1",
     "mocha": "^5.2.0",
-    "node-fetch-polyfill": "^2.0.6",
+    "node-fetch": "^2.6.0",
     "npm-run-all": "^4.1.5",
-    "ts-jest": "^21.2.4",
+    "ts-jest": "^24.1.0",
     "tslint": "^5.15.0",
     "tslint-react": "^3.6.0",
-    "typescript": "~2.9.2",
-    "whatwg-fetch": "^2.0.4"
+    "typescript": "~3.2.0",
```

Comment:
We need this version to be compatible with the polyfill we're using and that polyfill matches the web api.

Rest of the bumps here are to get to latest/over ts major rev

Reply:
can we bump this to latest? internally we're starting to use typescript 3.7.3

Reply:
We're trying to be conservative since we we want to keep as wide of a compatibility window as possible